snuba/web/db_query.py

```diff
@@ -476,7 +476,20 @@ def raw_query(
 
                 logger.exception("Error running query: %s\n%s", sql, cause)
             stats = update_with_status(QueryStatus.ERROR)
-        raise QueryException({"stats": stats, "sql": sql}) from cause
+        raise QueryException(
+            {
+                "stats": stats,
+                "sql": sql,
+                "experiments": clickhouse_query.get_experiments(),
```

Comment:
I don't think we should just arbitrarily change the structure of data we are returning to the client here unless we have really strong reasons for doing that.

Reply:
It certainly makes writing some tests easier.

On a more serious note, I think we're going to want to know which experiments ran on a particular query. One way to get that is from the tags, but if we're debugging locally then those aren't helpful. I think if these experiments continue to be used past this quarter, we should add them to the querylog itself. For now adding them to the response payload seems like a good compromise. Also this payload is only for debug queries, so it will largely be ignored.

Reply:
It is ok, they would only show up if snuba is in debug mode.
https://github.com/getsentry/snuba/blob/master/snuba/web/views.py#L456-L457